### ISSUE TITLE
Query Gate Linking Breaks New Hinges

### ISSUE THREAD

**Issue by talrey** [tsunamayo/Skywanderers#1046]
### Short Description of the issue:

Linking the new style of hinge to a Query gate causes the base of the hinge (the outside parts) to disappear, and the hinge stops working.

### Steps to reproduce the issue:

Place the new style of hinge down (the flat, rounded one). Place a Query gate. Use the Link Tool to link from the hinge to the Query.

### Environment
- Skywanderers version: 18w50b (Experimental branch)
- Graphics Card: Nvidia GTX 970

### Additional information:

I haven't been able to place blocks on the new hinges either.

[output_log.txt](https://github.com/tsunamayo/Skywanderers/files/2688353/output_log.txt)

**Comment by tsunamayo** [comment]
Hi, so I cant reproduce but there is definitely something wrong looking at your logs. Could you please send me the blueprint in which you get the issue? Do you reproduce all the time? Thanks!

**Comment by talrey** [comment]
I just tested it again to get some pictures and all that, but I can't seem to get it to happen now. It was happening 100% of the time when I reported, but now it's gone. Looks like there was a patch - I'm on 18w51a now - so I guess it's fixed, whatever you did! :laughing: 

By the way, I noticed while setting up a test for this bug that Query gates say "no system data" when linked to the new hinges. I guess that's to be expected for now though, since they're new tech and the logic system's still migrating over in some ways.

Anyway, issue is gone with the update so thanks :+1:
